src/firefly/js/visualize/ui/ColorTableDropDownView.jsx

```diff
@@ -261,34 +270,38 @@ const AdvancedColorPanel= ({allowPopout}) => {
                            hasCheckBox={true} checkBoxOn={useBlue}
                            onClick={() => changeBiasContrastColor(colorTableId,bias,contrast,useRed,useGreen,!useBlue)}/>}
             <DropDownVerticalSeparator useLine={true}/>
-            <div style={{width: 230, height: 126 * plotState.getBands().length}}>
+            <Box style={{width: 230, height: 126 * plotState.getBands().length}} pb={1}>
                         {
                             plotState.getBands().map( (b, idx) => (
-                                <Fragment key={b.key}>
-                                    <div style={{display:'flex', flexDirection: 'column', alignItems: 'center'}}>
-                                        <div style={{margin: `${idx?'40':'5'}px 0 0 0`, paddingTop:idx?5:0, textAlign:'center', alignSelf:'stretch', borderTop:`1px solid rgba(0,0,0,${idx?'.1':'0'}`}}>
-                                            <span style={{color:b.key}}>{b.key}</span> Bias
+                                <Box key={b.key}>
+                                    <Stack alignItems='center' spacing={0}>
+                                        <div style={{margin: `${idx?'10':'5'}px 0 0 0`, paddingTop:idx?3:0, textAlign:'center', alignSelf:'stretch', borderTop:`1px solid rgba(0,0,0,${idx?'.1':'0'}`}}>
+                                            <Typography level='body-xs' style={{padding: '5px 0 2px 0'}}>
+                                                <Typography level='body-xs' style={{color:b.key}}>{b.key}</Typography> Bias
+                                            </Typography>
                                         </div>
                                         <RangeSliderView {...{
-                                            style:{paddingTop: 7, width: 200},
+                                            style:{paddingTop: 0, marginTop:-12, width: 200},
```

Comment:
passing sx would be `sw:{pt:0, mt:-3}`

Reply:
I found that this actually works with `mt:-1.5` (12/8) which means it's based on a root font size of 8px.`mt:-3` would work for a root font-size of 4px. I think once we decide on and set a font-size at root, we may need to make changes to some of these `sx` values later? (unless we decide to go with 8px)